mappings/net/minecraft/block/Block.mapping

```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/class_2248 net/minecraft/block/Block
-	COMMENT A block is a voxel in a {@linkplain World world}.
-	COMMENT This class and its subclasses define all logics for those voxels.
+	COMMENT A block is a voxel in a {@linkplain World world}. {@link AbstractBlock},
+	COMMENT this class and its subclasses define all logic for those voxels.
```

Comment:
```suggestion
	COMMENT this class, and its subclasses define all logic for those voxels.
```

Reply:
It [works both with and without](https://en.wikipedia.org/wiki/Serial_comma) (unless you follow a specific style guide, which we don't).

Reply:
Personally I prefer the Oxford comma to be present

Reply:
think american english requires the comma before and unless it's only two items, as seen in americn college test

Reply:
It's Oxfordised now.